runtime/gc_glue_java/RealtimeGCDelegate.cpp

```diff
@@ -0,0 +1,697 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corp. and others
```

Comment:
should be `Copyright (c) 2019, 2019 IBM Corp. and others`... here and for all new files